### DISCUSSION TITLE
Zyxel 325v2 stuck on 2.82 (14160)

### DISCUSSION THREAD

**Discussion by bamse2005** [transmission/transmission#3990] [Q&A]
Greetings, is there a new Transmission (2.82) version compatible with Zyxel NAS 325v2 that I can upgrade to? Appreciate any pointers

**Comment by nevack** [comment]
I do not know specific instructions for your router, but I think you should use some firmware with Entware/OPKG support.
Transmission is available in Entware repos, 3.00 at the moment.

**Comment by bamse2005** [comment]
Has anyone manage to upgrade their Network Adaptor Storage (NAS) Zyxel (325v2/326) from Transmission 2.82 to higher one? Any advice for a n00b would be appreciated :)
